content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -736,15 +736,25 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-{{< feature-state for_k8s_version="v1.0" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="deprecated" >}}
 
 {{< caution >}}
-This admission controller plugin is **outdated** and **incomplete**, it may be
-unusable or not do what you would expect. It was originally designed to prevent
-the use of some, but not all, security-sensitive fields. Indeed, fields like
-`privileged`, were not filtered at creation and the plugin was not updated with
-the most recent fields, and new APIs like the `ephemeralContainers` field for a
-Pod.
+The Kubernetes project recommends that you **do not use** the
+`SecurityContextDeny` admission controller.
+
+This admission controller plugin is deprecated and disabled by default. If you
```

Comment:
nit: replace `This` with
The `SecurityContextDeny` admission controller plugin is ...

Can you link to more information about this plugin?

Reply:
I replaced this with an explicit name.

You can find a link with a bit more info down:
> For more historical context on this plugin, see [The birth of PodSecurityPolicy](https://kubernetes.io/blog/2022/08/23/podsecuritypolicy-the-historical-context/#the-birth-of-podsecuritypolicy) from the Kubernetes blog article about PodSecurityPolicy and its removal. The article details the PodSecurityPolicy historical context and the birth of the securityContext field for Pods.

We don't have much more documentation about this.